orchestra/actions/clone.py

```diff
@@ -11,10 +11,12 @@ def __init__(self, component, repository, config):
 
     @property
     def script(self):
+        script = 'mkdir -p "$(dirname "$SOURCE_DIR")"\n'
```

Comment:
If you want this here, make sure we don't have this in `orchestra` configuration.

Reply:
Note that this is not creating the source dir itself, rather it's ensuring that `$SOURCE_DIR` parent directory exists before invoking the configure script.
Anyway, I quickly did `grep -R mkdir` and saw no places where we create `$SOURCE_DIR` or its parent; this is because with our configuration the sources are either cloned directly by orchestra or they are extracted by the `extract.sh` helper which creates the target directory itself.